libraries/AP_Logger/AP_Logger.h

```diff
@@ -343,6 +343,7 @@ class AP_Logger
         AP_Int8 log_replay;
         AP_Int8 mav_bufsize; // in kilobytes
         AP_Int16 file_timeout; // in seconds
+        AP_Int16 min_MB_free;
```

Comment:
in Copter and Rover land we would normally put the "min" at the end of the variable name it's not a blocker.